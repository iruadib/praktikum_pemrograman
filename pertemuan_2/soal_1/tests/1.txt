Khoirul Adib Fairuza
100000000000000 123412341234 123123