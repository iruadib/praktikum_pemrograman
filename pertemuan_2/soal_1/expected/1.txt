Payslip for Employee
---------------------
Name: Khoirul Adib Fairuza
Tax (20%): Rp20.000.000.000.000,00
Installment: Rp123.412.341.234,00
Insurance: Rp123.123,00
Net Salary: Rp79.876.587.535.643,00
